packages/core/src/components/Search/Search.tsx

```diff
@@ -116,8 +118,10 @@ const Search = forwardRef(
         inputRole={searchResultsContainerId ? "combobox" : undefined}
```

Comment:
I think while trying to keep the functionality of the original Search I made a mistake here
the input should not have a searchbox role, the wrapper should not have a role at all, it doesn't make sense 🤦🏼